src/Sylius/Bundle/AdminBundle/Resources/config/routing/security.yml

```
@@ -17,3 +17,13 @@ sylius_admin_login_check:
 sylius_admin_logout:
     path: /logout
     methods: [GET]
+
+sylius_admin_request_password_reset:
+    path: /forgotten-password
```

```suggestion
    path: /reset-password
```
why no simple reset password?

---

@AdamKasp The same route was on shop side, I wanted stick to convention.